bin/run-tests.js

```diff
@@ -90,9 +90,6 @@ function generateBuiltTests() {
   testFunctions.push(function() {
     return run(common + '&nolint=true');
   });
-  testFunctions.push(function() {
-    return run(common + '&dist=min&prod=true');
```

Comment:
Hmm, this reminds me, once upon a time we had legit issues when minified so we started running our test suite against the minified assset.

I'm not 100% sure that is still a valid concern/issue, but it warrants further thought...

Reply:
sounds like a valid concern, but as you said Ember.js is no longer controlling its own minification process, so I'm not sure it's helpful to run the tests against just that single version of `terser` while out in the wild other versions and possibly even other minifiers are being used 🤔

Reply:
Yep, totally agree.